index.html

```diff
@@ -0,0 +1,168 @@
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about">
+        <div class="about-me">
+            <h2>About Me</h2>
+            <p> I can help you build a product, feature or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded,
+                don't hestiate to contact me.</p>
+            <div class="resume-btn"><a href="#">Get My Resume</a></div>
+        </div>
+        <div class="about-me-skils">
+            <div class="worklanguages">
+                <img src="./images/icon-languages.svg" alt="Image depicting languages">
+                <h2>LANGUAGES</h2>
+                <div class="lang-tags">
+                    <ul>
+                       <li>HTML</li>
+                       <li>Ruby</li>
+                       <li>JavaScript</li>
+                       <li>CSS</li>
+                    </ul>
+                </div>
+            </div>
```

Comment:
- [ ] For the screen size of "375 px," as shown in the design, there are some differences in how your card is displayed compared to the design. I recommend making the necessary adjustments to your code to ensure that the card closely matches the design. Pay attention to the spacing, alignment, font styles, and overall layout of the card. By carefully reviewing the design and making the necessary changes, you can ensure that the card appears as close as possible to the intended design on a screen size of "375 px."

![image](https://github.com/zhorabay/My-portfolio/assets/55840999/7bf2e5a7-0548-4d40-a17e-67829153b198)